src/client/dfuse/ops/setattr.c

```diff
@@ -63,7 +63,6 @@ dfuse_cb_setattr(fuse_req_t req, struct dfuse_inode_entry *ie, struct stat *attr
 	if (to_set & FUSE_SET_ATTR_ATIME) {
 		DFUSE_TRA_DEBUG(ie, "atime %#lx", attr->st_atime);
 		to_set &= ~(FUSE_SET_ATTR_ATIME | FUSE_SET_ATTR_ATIME_NOW);
-		dfs_flags |= DFS_SET_ATTR_ATIME;
```

Comment:
Only comment here is we may in theory get to L109 and have dfs_flags being 0 so perhaps we could optimise just call stat rather than setattr there.  It seems it's unlikely to happen but adding a D_ERROR call for that case would be no bad thing if you're pushing again.

Reply:
so dfs_osetattr will do a stat, and if flags is empty it just doesn't do anything. so we are good.